### DISCUSSION TITLE
[001260] dandiset validation: about.0: {}

### DISCUSSION THREAD

**Discussion by bendichter** [dandi/helpdesk#171] [Dandiset issues]
This dandiset is not validating and therefore cannot be published.

https://dandiarchive.org/dandiset/001260

> about.0: {} is not valid under any of the given schemas

<img width="1417" alt="image" src="https://github.com/user-attachments/assets/04d31e50-0456-4a91-9654-5c06e4de41ef">

It looks like this may be an issue with "Subject Matter"

@PierreLeMerre, are you able to fix this in the metadata editor?

**Comment by PierreLeMerre** [comment]
Hi Ben,

We are just holding onto the data until the paper is actually accepted.

But the data is ready to be pushed (see attachement 😉)

Best wishes,


Pierre Le Merre | PhD

Research Specialist
Carlén Lab
Department of Neuroscience | Karolinska Institutet
Solnavägen 9  | 171 65 Solna, Sweden



***@***.***
De : Ben Dichter ***@***.***>
Date : lundi, 25 novembre 2024 à 16:37
À : dandi/dandi-archive ***@***.***>
Cc : Pierre le Merre ***@***.***>, Mention ***@***.***>
Objet : [dandi/dandi-archive] dandiset validation: about.0: {} (Issue dandi/helpdesk#170)
Vous n’obtenez pas souvent d’e-mail à partir de ***@***.*** Pourquoi c’est important<https://aka.ms/LearnAboutSenderIdentification>

This dandiset is not validating and therefore cannot be published.

https://dandiarchive.org/dandiset/001260

about.0: {} is not valid under any of the given schemas
image.png (view on web)<https://github.com/user-attachments/assets/04d31e50-0456-4a91-9654-5c06e4de41ef>

It looks like this may be an issue with "Subject Matter"

@PierreLeMerre<https://github.com/PierreLeMerre>, are you able to fix this in the metadata editor?

—
Reply to this email directly, view it on GitHub<https://github.com/dandi/helpdesk/issues/170>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AMA5E343444JT3YA5KTQGYL2CM73ZAVCNFSM6AAAAABSOI6OCKVHI2DSMVQWIX3LMV43ASLTON2WKOZSGY4TCMJYHE2DANI>.
You are receiving this because you were mentioned.Message ID: ***@***.***>



När du skickar e-post till Karolinska Institutet (KI) innebär detta att KI kommer att behandla dina personuppgifter. Här finns information om hur KI behandlar personuppgifter<https://ki.se/om-ki/integritetsskyddspolicy>.


Sending email to Karolinska Institutet (KI) will result in KI processing your personal data. You can read more about KI’s processing of personal data here<https://staff.ki.se/data-protection-policy>.

**Comment by waxlamp** [comment]
@bendichter, as this is an issue with the metadata in a specific Dandiset, and not an issue with the archive itself, I'm going to go ahead and close this. Can you track this as a TODO somewhere else?

**Comment by satra** [comment]
@bendichter - adjusted the about issue - there was an empty subject matter entry. however, the names are not properly formatted.

**Comment by PierreLeMerre** [comment]
As mentioned above, no data has currently been pushed to this. I am holding on the NWB files until publication of our study. I would have liked to embargo the dataset but we are Europe based and the embargo option was asking some NIH related info I couldn't provide. 

The data is ready to be uploaded and has passed the nwbinspector test with 0 issues. Let's talk back here if there is still a problem when actual data populate the archive.

Best,

**Reply by bendichter** [reply]
Yes but there is a problem with the metadata that will still be a problem when you upload data

**Reply by PierreLeMerre** [reply]
Ok. I see. sorry for the misunderstanding. I am checking the issue now.

**Reply by bendichter** [reply]
It's fine. I was mostly complaining that the previous validation errors were difficult to decipher

**Reply by PierreLeMerre** [reply]
Should be better now. Only the no file error is left.

**Reply by bendichter** [reply]
@PierreLeMerre great! Could you please change the title of the preprint related resource from "Preprint" to the title of the paper?
